avocado/utils/multipath.py

```diff
@@ -106,6 +106,37 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "multipathd show maps format '%n %w'"
+    wwids = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
+    for wwid in wwids.splitlines(): 
+        if mpath in wwid:
+            return wwid.split()[1]
+
+
+def is_mpath(mpath):
```

Comment:
Just a small suggestion on a better function name: `is_mpath_dev` ?

Reply:
Changed the function name to is_mpath_dev.